### PR TITLE
Add reference paper

### PR DESCRIPTION
**Pull request by delta2323**

### CODE REVIEW

**Comment by unnonouno** [comment]
Write bibtex

**Comment by delta2323** [comment]
I added bibtex.

File: chainer_bibtex.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@inproceedings{chainer_learningsys2015,
+  author       = "Seiya Tokui, Kenta Oono, Shohei Hido and Justin Clayton",
```
Comment (unnonouno) [inline chainer_bibtex.txt]: "Tokui, Seiya and Oono, Kenta and Hido, Shohei and Clayton, Justin"

File: chainer_bibtex.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  title        = "Chainer: a Next-Generation Open Source Framework for Deep Learning",
+  booktitle    = "Neural Information Processing Systems(NIPS), Workshop on Machine Learning Systems(LearningSys)",
```
Comment (unnonouno) [inline chainer_bibtex.txt]: Maybe "Proceedings of ..."

**Comment by unnonouno** [comment]
LGTM

**Pull request merged**